py/desiperf/instperfapp/data_mgt/data_general.py

```diff
@@ -0,0 +1,77 @@
+
+import pandas as pd
+import numpy as np
+import os, glob
+
+from astropy.table import Table
+from astropy.time import Time
+import psycopg2
+from datetime import datetime
+from bokeh.models import ColumnDataSource
+
+
+class DataSource():
+    # Currently this only works on the DESI Server
```

Comment:
Future upgrade: Add a try/except loop, probably at psycopg2.connect(), to handle cases where someone attempts to run a DB query when not on the DESI server

Reply:
I will put this in now and re-commit. I can also add a message to alert the user why it won't work for them.